<launch>
    <param name="/use_sim_time" value="true" />
    <include file="$(find turtlebot_bringup)/launch/minimal.launch"/>
    <include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch">
    <arg name="align_depth" value="true" />
    </include>
    
    
    <node pkg="urg_node" type="urg_node" name="laser" />
    <node pkg="tf" type="static_transform_publisher" name="scan_broadcaster" args="0.1 0 0 0 0 0 1 base_footprint laser 100" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find grp-rouge)/rviz/challenge3_tbot.rviz"/>
    <node name="gmapping" pkg="gmapping" type="slam_gmapping"/>
    
    <node pkg="grp-rouge" type="markbottle.py" name="mark" output="screen" />
    <node pkg="grp-rouge" type="move_turtlebot.py" name="move">
    	<remap from="/cmd_vel_mux/input/navi" to='/cmd_vel_mux/input/navi'/>
    </node>
    <node pkg="grp-rouge" type="detectbottle.py" name="detection"/>
</launch>

